src/core/LabeledInput/LabeledInput.tsx

```diff
@@ -79,7 +92,9 @@ export const LabeledInput = React.forwardRef<
         {
           'iui-disabled': disabled,
           [`iui-${status}`]: !!status,
-          [`iui-${displayStyle}`]: displayStyle !== 'default',
+          'iui-inline-label': displayStyle === 'inline',
+          'iui-with-message': !!message && displayStyle !== 'inline',
```

Comment:
Don't our hybrid cases support having message with inline styling?

Reply:
Where message should appear then?

Reply:
Like in one of these, no?
![image](https://user-images.githubusercontent.com/36186912/133742327-2e278a65-7fbf-41c6-93ec-fd65f0fdcafd.png)

Reply:
It works,
![image](https://user-images.githubusercontent.com/81580355/133742945-687e039e-fdb5-43cf-80b2-03b7a9ac7053.png)

Reply:
oh wait, need inline label..

Reply:
I think I know what Mayank had in mind here.. He didnt do that message could appear as tooltip, so didnt want to display it at all, if its inline type...

Reply:
So I removed inline need for the message. Tests were not updated, so they are failing. But now I am concerned, what we shold do. This is kinda a breaking change...

Reply:
My initial intent was to preserve current behavior for `displayStyle: inline`, and then add a new prop for `messageDisplayStyle`.

Reply:
Reverted to preserve current behavior and prevent breaking visual change.